lib/services.js

```diff
@@ -2,7 +2,12 @@ var fs = require('fs');
 var ejs = require('ejs');
 
 ejs.filters.q = function(obj) {
-  return JSON.stringify(obj, null, 2 );
+  // to handle strings being returned with double-quotes
+  // but linter requires single quotes for strings
+  if (typeof obj == 'string' && !/\\'/.test(obj))
+    return '\'' + obj + '\'';
+  else
+    return JSON.stringify(obj, null, 2);
```

Comment:
not sure if this is the best approach
I have modified the `ejs` filter so string gets wrapped in single quote instead of double quote that JSON.stringify does